codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -471,14 +479,12 @@ void writeAllocatedBytes() {
         /**
          * Reset the number of bytes that have been allocated to this stream by the priority algorithm.
          */
-        void resetAllocated() {
+        private void resetAllocated() {
```

Comment:
I think so. The intended visibility of this method is "private" i.e. only accessible internal to this class.  I prefer to construct classes as if they were no inner classes, unless there is a reason not to (i.e. they method is actually called outside scope and we want to avoid the possibility of synthetic method generation).